### PR TITLE
return info about status of played audio file;

### PR DESCRIPTION
**Pull request by hugodeco**

### CODE REVIEW

**Comment by Deantwo** [comment]
I merged #187, so please updated this to match.

File: Asterisk.2013/Asterisk.NET/FastAGI/AGIScript.cs [hunk 1]
```diff
@@ -275,10 +275,13 @@ protected internal void SetPriority(string label)
...
 		/// </summary>
 		/// <param name="file">name of the file to play.</param>
-		protected internal void StreamFile(string file)
+		protected internal int StreamFile(string file)
 		{
-			this.Channel.SendCommand(new Command.StreamFileCommand(file));
-		}
+            AGIChannel channel = this.Channel;
+            channel.SendCommand(new Command.StreamFileCommand(file));
+            return channel.LastReply.ResultCode;
```
[the remark this answers is not in the document]
Comment (hugodeco) [reply Asterisk.2013/Asterisk.NET/FastAGI/AGIScript.cs]: I'm not sure about it. But, i used same lines of another existent functions(ControlStreamFile). I Had many positive updates on my application and that is a high flow app an we use multi threads. 

Não tenho certeza, porém usei o mesmo código descrito nas outras funções, o resultado foi bem positivo na minha aplicação e minha aplicação é de alto fluxo e com várias threads.

[the remark this answers is not in the document]
Comment (Deantwo) [reply Asterisk.2013/Asterisk.NET/FastAGI/AGIScript.cs]: Yeah it was just changed in #187, so please update it to make it similar to those.

**Comment by Deantwo** [comment]
Recreated as #211.

**Pull request closed without merging**